Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ.yml

```diff
@@ -0,0 +1,613 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SailPointIdentityIQ
+  version: -1
+configuration:
+- additionalinfo: The FQDN/IP of the IdentityIQ server to connect
+  display: IdentityIQ Server URL (e.g. https://identityiq-server.com/identityiq)
+  name: identityiq_url
+  required: true
+  type: 0
+- defaultvalue: OAUTH
+  display: Authentication Type
+  hidden: false
+  name: auth_type
+  options:
+  - OAUTH
+  - BASIC
+  required: true
+  type: 15
+- additionalinfo: Client Id for OAuth 2.0
+  display: Client Id (for OAuth 2.0)
+  hidden: false
+  name: client_id
+  required: false
+  type: 4
+- additionalinfo: Client Secret for OAuth 2.0
+  display: Client Secret (for OAuth 2.0)
+  hidden: false
+  name: client_secret
+  required: false
+  type: 4
+- additionalinfo: Username for Basic Authentication
+  display: Username (for Basic Auth)
+  hidden: false
+  name: username
+  required: false
+  type: 0
+- additionalinfo: Password for Basic Authentication
+  display: Password (for Basic Auth)
+  hidden: false
+  name: password
+  required: false
+  type: 4
```

Comment:
It is highly recommended to use type: 9 (credentials) for `username/password`.
This will allow users to manage credentials ([see article](https://xsoar.pan.dev/docs/reference/articles/managing-credentials)).

It will also be worth your time to include the Oauth configuration under this parameter - i.e. for this parameter to accept either `username/password` or `client_id/client_secret`, though the UX for it will not be as nice as the second field (password field) has a display name of "Password".

Reply:
Removed basic auth altogether. Only OAUTH 2.0 (Client Id & Secret) is supported.